integration/operate-first/README.md

```diff
@@ -0,0 +1,92 @@
+
+### Logging In 
+You can access the Smaug cluster with this [OpenShift console login link](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/authorize?client_id=console&redirect_uri=https%3A%2F%2Fconsole-openshift-console.apps.smaug.na.operate-first.cloud%2Fauth%2Fcallback&response_type=code&scope=user%3Afull&state=98ae2ceb). Click on `operate-first` to login with GitHub authentication. Once logged into the OpenShift console, you can use [this link](https://oauth-openshift.apps.smaug.na.operate-first.cloud/oauth/token/display) to get an `oc login` command with a token that will let you login to the Smaug OpenShift cluster from your terminal. 
+
+## Deploying Fybrik Cluster-Scoped Resources
+This `integration/operate-first` directory contains the raw YAML files of the cluster scoped resources deployed for Fybrik. These files are generated from the Helm charts in `charts/fybrik`.
+
+If the Helm chart has been updated, follow the steps below to generate the new yaml files:
+1. Install yq and Helm in this repo. Run these commands from the root directory of this repo:
+```bash
+cd hack/tools
+./install_yq.sh
+./install_helm.sh
+```
+2. Go back to the `integration/operate-first` folder and set up the Python environment there
+```bash
+cd samples/operate-first
+pipenv install
+pipenv shell
+```
+3. Run Makefile to generate new YAML files from the Helm charts:
+```bash
+make all
+```
+
+After the cluster-scoped YAML files are generated, create a PR to the [operate-first/apps](https://github.com/operate-first/apps) repository in the [cluster-scope/base](https://github.com/operate-first/apps/tree/master/cluster-scope/base) directory with the YAML files in the subdirectories organized by resource type. More documentation about contributing to the `operate-first/apps` repository can be found [here](https://github.com/operate-first/apps/tree/master/docs/content)
+
+## Deploying Namespace-Scoped resources
+Operate First has an [ArgoCD](https://argo-cd.readthedocs.io/en/stable/) instance deployed on MOC that can be used to deploy OpenShift resources located on a Git Repository. Fybrik has been onboarded to ArgoCD by following [these instructions](https://github.com/operate-first/apps/blob/master/docs/content/argocd-gitops/onboarding_to_argocd.md) and an ArgoCD project has been created for Fybrik. You can login to [the ArgoCD instance](https://argocd.operate-first.cloud/applications?proj=&sync=&health=&namespace=&cluster=&labels=) with the same login method as above. We have deployed 3 [ArgoCD applications](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#applications) which are automatically synced with the latest release of Fybrik. See the ArgoCD applications below pointing to the corresponding Fybrik repositories:
+* fybrik -> https://github.com/fybrik/charts
+* arrow-flight-module -> https://github.com/fybrik/arrow-flight-module
+* vault -> https://github.com/fybrik/fybrik
```

Comment:
Vault can also be synced against latest release with URL https://github.com/fybrik/charts and `charts/vault` path

Reply:
@revit13 When I set the vault ArgoCD application to point to [https://github.com/fybrik/charts](https://github.com/fybrik/charts), I get this error in the notebook sample running on the Operate First JupyterHub instance: 

```
FlightServerError                         Traceback (most recent call last)
<ipython-input-17-3b17ce255ef3> in <module>
      1 # Send request and fetch result as a pandas DataFrame
----> 2 info = client.get_flight_info(fl.FlightDescriptor.for_command(json.dumps(request)))
      3 reader: fl.FlightStreamReader = client.do_get(info.endpoints[0].ticket)
      4 df: pd.DataFrame = reader.read_pandas()

/opt/app-root/lib64/python3.8/site-packages/pyarrow/_flight.pyx in pyarrow._flight.FlightClient.get_flight_info()

/opt/app-root/lib64/python3.8/site-packages/pyarrow/_flight.pyx in pyarrow._flight.check_flight_status()

FlightServerError: When listing objects under key 'data.csv' in bucket 'notebook-sample': AWS Error [code 15]: Access Denied with address : 169.50.118.97 with address : 169.50.118.97. Detail: Python exception: OSError
```
I changed the Vault ArgoCD application manifest back to point to [https://github.com/fybrik/fybrik](https://github.com/fybrik/fybrik) and the notebook sample works as expected. 

I also noticed vault in the charts repo deploys the `vault-secrets-manager` cluster role but vault in the fybrik repo does not. We can't deploy cluster roles or cluster role bindings using ArgoCD.

Reply:
@revit13 The above error is now resolved since @roytman updated the [https://github.com/fybrik/charts]( https://github.com/fybrik/charts) repo. I changed the documentation and vault ArgoCD app to point to `fybrik/charts`